pori_python/ipr/summary.py

```diff
@@ -342,6 +364,66 @@ def section_statements_by_genes(
     return genes
 
 
+def prep_single_ipr_variant_comment(item):
+    cancer_type = ",".join(item["cancerType"])
+    if not cancer_type:
+        cancer_type = "no specific cancer types"
+    cancer_type = f" ({cancer_type})"
+    section = [f"<h2>{item['variantName']}{cancer_type}</h2>"]
+    section.append(f"<p>{item['text']}</p>")
+    return section
+
+
+def ipr_analyst_comments(
```

Comment:
So, on report creation, we want to pre-fill the comment section with relevant comments from previous reports? Can we add a docstring explaining it?